examples/nvexec/maxwell_distributed.cpp

```diff
@@ -478,13 +478,13 @@ int main(int argc, char *argv[]) {
 
   for (std::size_t compute_step = 0; compute_step < n_iterations; compute_step++) {
     auto compute_h = ex::when_all(
-      ex::just() | exec::on(gpu, ex::bulk(bulk_cells, bulk_h_update)),
-      ex::just() | exec::on(gpu_with_priority, ex::bulk(border_cells, border_h_update))
+      ex::just() | exec::starts_on(gpu, ex::bulk(bulk_cells, bulk_h_update)),
+      ex::just() | exec::starts_on(gpu_with_priority, ex::bulk(border_cells, border_h_update))
```

Comment:
code that uses `exec::on` (as opposed to `stdexec::on`) is expecting there-and-back-again behavior, so it should not be changed to `starts_on`. here and elsewhere.

Reply:
Thanks, good catch. I've removed these (and related) changes in the last batch of changes.